avocado/utils/kernel.py

```diff
@@ -32,16 +33,19 @@ class KernelBuild(object):
     Build the Linux Kernel from official tarballs.
     """
 
-    URL = 'https://www.kernel.org/pub/linux/kernel/v3.x/'
+    URLS = ['https://www.kernel.org/pub/linux/kernel/v3.x/',
+            'https://www.kernel.org/pub/linux/kernel/v4.x/']
+
     SOURCE = 'linux-{version}.tar.gz'
 
-    def __init__(self, version, config_path=None, work_dir=None,
+    def __init__(self, version, config_path=None, mirrors=None, work_dir=None,
```

Comment:
Please don't change the order of the arguments. Add your new argument at the end.